INSTALL.md

```diff
@@ -38,22 +38,25 @@ https://github.com/mauricext4fs/docker-proxy.git
    This is optional... if you do not need to use a remote backend 
    then leave this section commented.
 
-5. Run the configure script "./configure" which will do the following: 
+5. Go to .env file and replace "some_project_name" value of variable COMPOSE_PROJECT_NAME 
```

Comment:
Nope... the point of that project is the user should have 0 knowledge of Docker. We are already making them enter a shit load of value in docker-compose.yml manually. If you want to create a .env then please move all the value in .env at the very least. There is another PIV regarding the .env using configure. You should generate .env from the response from the user during configure. There should be no need for the user to change values in any files.